pkg/mimir/modules.go

```diff
@@ -422,6 +424,7 @@ func (t *Mimir) initRuntimeConfig() (services.Service, error) {
 	t.Cfg.StoreGateway.ShardingRing.KVStore.Multi.ConfigProvider = multiClientRuntimeConfigChannel(t.RuntimeConfig)
 	t.Cfg.QueryScheduler.ServiceDiscovery.SchedulerRing.KVStore.Multi.ConfigProvider = multiClientRuntimeConfigChannel(t.RuntimeConfig)
 	t.Cfg.OverridesExporter.Ring.Common.KVStore.Multi.ConfigProvider = multiClientRuntimeConfigChannel(t.RuntimeConfig)
+	// TODO probably need to add for continuous-test
```

Comment:
You mean use a ring to ensure only a single `continuous-test` instance is running?

Reply:
I don't actually know when I added that TODO or what my thought was. Because multiple continuous-test should be fine to run at the same time. I'll remove.